package.json

```diff
@@ -1,19 +1,20 @@
 {
-  "name": "react-native-signature-capture",
-  "version": "0.4.11",
+  "name": "wix-react-native-signature-capture",
+  "version": "1.0.0",
   "description": "Lets users sign their signatures",
   "main": "SignatureCapture.js",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/RepairShopr/react-native-signature-capture.git"
+    "url": "https://github.com/wix-playground/react-native-signature-capture"
```

Comment:
This will change the project repository of the npm?
[npm](https://www.npmjs.com/package/react-native-signature-capture)